### PR TITLE
quickemu: 4.9.4-unstable-2024-05-28 -> 4.9.5

### LINKED ISSUE #322484: Update request: quickemu 4.9.4 → 4.9.5
- Package name: quickemu
- Latest released version: 4.9.5

- Current version on the unstable channel: 4.9.4
- Current version on the stable/release channel: 4.9.5

- [X] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**

fedx-sudo@pm.me
martin@wimpress.org
-----

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by pyrox0**
## Description of changes
Fixes #322484, also adds the changelog to the package file.
Changelog: https://github.com/quickemu-project/quickemu/releases/tag/4.9.5

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by mjoerg — COMMENTED** [review]
The source needs to be updated:
```nix
    rev = "refs/tags/${finalAttrs.version}";
    hash = "sha256-UlpNujF2E8H1zcWTen8D29od60pY8FaGueviT0iwupQ=";
```

New dependencies were added, see https://github.com/quickemu-project/quickemu/blob/4.9.5/package.nix#L32-L54.

`fetchpatch2` is no longer in use and can be removed.

**Review by flexiondotorg — CHANGES_REQUESTED** [review]
Please see what @mjoerg said. As for dependency update you can use my local nix package as reference, I kept in it step with my upstream changes:

- <https://github.com/wimpysworld/nix-config/blob/main/pkgs/quickemu/default.nix>

**Comment by pyrox0** [comment]
Updating following both of your suggestions!

**Review by flexiondotorg — APPROVED** [review]

**Review by flexiondotorg — APPROVED** [review]

File: pkgs/by-name/qu/quickemu/package.nix [hunk 1]
```diff
@@ -51,18 +51,22 @@ let
     xdg-user-dirs
```
[the remark this answers is not in the document]
Comment (pyrox0) [reply pkgs/by-name/qu/quickemu/package.nix]: Done!

**Review by flexiondotorg — APPROVED** [review]

**Comment by etrigan63** [comment]
What's the hold up?

**Pull request merged**